charts/rancher-backup/2.0.0/Chart.yaml

```diff
@@ -5,7 +5,8 @@ annotations:
   catalog.cattle.io/namespace: cattle-resources-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/provides-gvr: resources.cattle.io.resourceset/v1
-  catalog.cattle.io/rancher-version: < 2.6.1-0
+  catalog.cattle.io/rancher-version: '>= 2.6.1-0 <= 2.6.100-0'
```

Comment:
this rancher-version should not have been modified, it's expected to be hidden after 2.6.0

kube-version looks fine.